### PR TITLE
[#32128] qa fixing sprint1

### PR DESCRIPTION
**Pull request by HierenFeb**

### CODE REVIEW

**Comment by kimpepper** [comment]
This page needs some fixes https://agov_pr552.agov.com.au/news-media

File: themes/agov/agov_whitlam/sass/_misc.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// Custome Css
+
+// Fixe weird of heading font
+.heading, h1, h2, h3, h4, h5, h6 {
+	font-weight: 400;
+}
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/sass/_misc.scss]: Not sure this is needed. I've got a separate PR that is fixing up fonts. #554

**Comment by HierenFeb** [comment]
@kimpepper  I just put some code into correct component, please check!

**Comment by JohnAlbin** [comment]
I'll take a look.

**Pull request merged**